src/main/java/pl/ateam/disasteralerts/web/SignupViewController.java

```diff
@@ -0,0 +1,14 @@
+package pl.ateam.disasteralerts.web;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/signup")
+public class SignupViewController {
+    @GetMapping
+    public String signup() {
```

Comment:
```suggestion
    public String signupView() {
```
tak jak przy indeksie nazwałaś metodę indexView()